apps/taskman/models.py

```diff
@@ -40,14 +41,30 @@ class JiraResolution(models.TextChoices):
     TEST_PENDING = "Test Pending"
 
 
+WORKFLOW_TO_JIRA_STATUS = {
+    WorkflowModel.OSIMState.NEW: (JiraStatus.NEW, JiraResolution.NONE),
+    WorkflowModel.OSIMState.TRIAGE: (JiraStatus.REFINEMENT, JiraResolution.NONE),
+    WorkflowModel.OSIMState.PRE_SECOND_ASSESSMENT: (
+        JiraStatus.REFINEMENT,
+        JiraResolution.NONE,
+    ),
+    WorkflowModel.OSIMState.SECOND_ASSESSMENT: (
+        JiraStatus.IN_PROGRESS,
+        JiraResolution.NONE,
+    ),
+    WorkflowModel.OSIMState.DONE: (JiraStatus.CLOSED, JiraResolution.DONE),
+    WorkflowModel.OSIMState.REJECTED: (JiraStatus.CLOSED, JiraResolution.WONT_DO),
+}
```

Comment:
I would suggest not to hard-code it but make it rather a part of the workflow definition as I drafted in
https://github.com/RedHatProductSecurity/osidb/pull/255/files
Or do you think there is reason why defining it in the code is better?